ios/browser/api/query_filter/query_filter_service.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_QUERY_FILTER_QUERY_FILTER_SERVICE_H_
+#define BRAVE_IOS_BROWSER_API_QUERY_FILTER_QUERY_FILTER_SERVICE_H_
+
+#import <Foundation/Foundation.h>
+
+OBJC_EXPORT
+@interface QueryFilterService : NSObject
```

Comment:
Does this actually need its own class, or can it be exported as a method on `URL`?

https://github.com/brave/brave-core/blob/master/ios/browser/api/url/url_utils.h#L14

Reply:
Yah sure I think I can do that

Reply:
Done